Jfrog platform 1.3.0

MahithaB opened the pull request:
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

estherk15 reviewed (commented):
Hi @MahithaB thanks for your work on this! I left some recommendations and suggestions. Let me know if you have any questions on this. There's room to format this documentation with tabs, similar to [Aerospike](http://localhost:1313/integrations/aerospike/?tab=host#configuration), I can help with that. Can you confirm this as the information outline:
FluentD Installation
  a. OS/Virtual Machine
      i. Gem based install
      ii. Configure FluentD
  b. Docker
  c. Kubernetes
  d. Artifactory, Artifactory-HA and Xray ENV variables (I think we can pull this out instead of repeating it multiple times.)

MahithaB commented on jfrog_platform_self-hosted/README.md:
````diff
@@ -0,0 +1,286 @@
+
+
+```text
+** For Gem based install, Ruby Interpreter has to be setup first, following is the recommended process to install Ruby
````
@estherk15 Gem based install is one of the suggested forms of installation just like CentOS/ MacOS etc. And configure fluentd section is common to any form of installation mentioned in the table above. That is the reason why we had Configure Fluentd as a sub section that is common to all OS/VM installations

estherk15 replied:
@MahithaB What if we move Configure Fluentd to the top and move Gem based install below that so readers can still access both. Having a header for Gem based install, gives users the option to link to it or share links to it.

MahithaB replied:
@estherk15 Configuring Fluentd step is something that has to be done after the OS/VM or Gem-based installation is done. So, moving it up might add confusion. Also gem based installation is a form of installation that can be done on any VM, so it being a part of OS / VM section makes it easier

estherk15 replied:
That makes sense! To confirm, the information structure is OS/Virtual Machine and both the Gem based install and Configure FluentD are subsections.
```
OS/Virtual Machine
- Gem based install
- Configure FluentD
```

MahithaB commented:
@estherk15
d. Artifactory, Artifactory-HA, and Xray ENV variables (I think we can pull this out instead of repeating it multiple times.)

The reason for not abstracting out the env variable section is that we've had feedback/customer support tickets in the past where users had difficulty setting environment variables because Artifactory-HA section for example doesn't have steps to configure .env vars under it

estherk15 commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+### Requirements
+
+* Your [Datadog API key][5].
+* If you have not installed the JFrog Platform (Self-hosted) tile yet, install the tile.
```
```suggestion
* Install the JFrog Platform (Self-hosted) integration.
```

estherk15 commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+#### JFrog Xray Logs dashboard
+Thi dashboard provides a summary of access, service and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight
+
+#### JFrog Xray Violations Dashboard
```
```suggestion
#### JFrog Xray Violations dashboard
```

estherk15 reviewed (commented):
Thanks for the edits @MahithaB! I found a few more edits for consistency. Can you also confirm you've seen the other comments/suggestions in the Files Changed tab, for example [this comment](https://github.com/DataDog/integrations-extras/pull/1895/files?file-filters%5B%5D=.csv&file-filters%5B%5D=.example&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.py&file-filters%5B%5D=.toml&file-filters%5B%5D=.txt&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&owned-by%5B%5D=estherk15&show-deleted-files=true&show-viewed-files=true#r1243800140)

MahithaB commented on jfrog_platform_self-hosted/README.md:
````diff
@@ -0,0 +1,286 @@
+
+```text
+
+For Datadog as the observability platform, execute these commands to setup the docker container running the fluentd installation
````
They're actually a continuous set of commands. So, makes sense to number them accordingly

MahithaB commented:
@estherk15 thank you for the review. I was just looking at the checks and I don't see any specific error in Validate Repository /run/validate (pull_request) check but it throws an error. Do I need to fix something on my end?

estherk15 commented:
> @estherk15 thank you for the review. I was just looking at the checks and I don't see any specific error in Validate Repository /run/validate (pull_request) check but it throws an error. Do I need to fix something on my end?



> @estherk15 thank you for the review. I was just looking at the checks and I don't see any specific error in Validate Repository /run/validate (pull_request) check but it throws an error. Do I need to fix something on my end?

Hi @MahithaB , this might be a configuration issue but I'm not as sure of this. After the other reviewers look through your PR, they may have better insight on this check.

nmuesch commented on jfrog_platform_self-hosted/CHANGELOG.md:
```diff
@@ -1,5 +1,10 @@
+All changes to the JFrog Platform (Self-hosted) integration will be documented in this file.
+
+## [1.3.0] - Jun 26th, 2023
+* Modified installation steps for send logs, metric and violations data to Datadog
```
nit:
```suggestion
* Modified installation steps for sending logs, metric and violations data to Datadog
```

nmuesch commented on jfrog_platform_self-hosted/manifest.json:
```diff
@@ -0,0 +1,81 @@
+      }
+    },
+    "dashboards": {
+      "JFrog Artifactory Logs (Self-hosted)": "assets/dashboards/jfrog_artifactory_logs_(self-hosted).json",
+      "JFrog Artifactory Metrics (Self-hosted)": "assets/dashboards/jfrog_artifactory_metrics_(self-hosted).json",
+      "JFrog Xray Logs (Self-hosted)": "assets/dashboards/jfrog_xray_logs_(self-hosted).json",
+      "JFrog Xray Metrics (Self-hosted)": "assets/dashboards/jfrog_xray_metrics_(self-hosted).json",
+      "JFrog Xray Violations (Self-hosted)": "assets/dashboards/jfrog_xray_violations_(self-hosted).json"
```
In order to provide a smooth experience for users, can you continue to use the same `key` as the old manifest file? 
Ex: `Jfrog Artifactory Dashboard`. 

Unfortunately we cannot remove existing dashboards or change the `key` at this time. However the changes to the dashboard.json payloads themselves are still OK!

nmuesch reviewed (changes requested):
Two quick comments from my review. Let me know if you have any questions!

estherk15 commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,274 @@
+
+## Setup
+
+`Note! You must follow the order of the steps throughout Datadog Configuration`
```
By the nature of numbered lists, it might not be necessary to include this. If you are seeing this as a common issue for customers:
```suggestion
Follow the instructions in the order listed:
```

estherk15 commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,274 @@
+
+Let JFrog know how we can improve the integration. Feel free to visit our GitHub for more detailed documentation at: [github.com/jfrog/log-analytics-datadog][4].
+
+### **IMPORTANT NOTE : The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will be deprecated by July 31st 2023.** 
```
Instead of placing the deprecation note here, recommend using an alert at the top of the page under Overview section above.
```suggestion
<div class="alert alert-warning">The JFrog Platform (self-hosted) check replaces the <a href="https://docs.datadoghq.com/integrations/jfrog_platform/">JFrog Platform check</a> which is deprecated. JFrog metrics are gathered by FluentD.</div>
```

MahithaB commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,274 @@
+## Overview
+
```
@estherk15 doesn't updating the README update https://docs.datadoghq.com/integrations/jfrog_platform as well? If that is the case, linking it might confuse the customers

MahithaB replied:
Instead, I can add 

<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will be deprecated by July 31st 2023.</div>

Because we're using Fluentd and the check that is in place right now (JFrog Platform (self-hosted)) check will be removed by the end of this month.

estherk15 replied:
Thanks for the clarification, I may have been confused on what is being deprecated! 
Suggested wording: "Agent check will deprecate on July 31, 2023."

MahithaB replied:
changed it, can you please review again

estherk15 replied:
Thanks @MahithaB, just made a suggestion for formatting, approved for documentation!

estherk15 commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,274 @@
+## IMPORTANT NOTE
```
```suggestion
```

estherk15 commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,274 @@
+## IMPORTANT NOTE
+**The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will deprecate on July 31st 2023**
```
Formatting for an alert div
```suggestion
<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. Agent check will deprecate on July 31st 2023.</div>
```

MahithaB commented:
@estherk15 thank you so much for the review, all the suggestions are implemented

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+#### JFrog Artifactory dashboard
+This dashboard is divided into three sections Application, Audit and Requests
+* **Application** - This section tracks Log Volume(information about different log sources) and Artifactory Errors over time(bursts of application errors that may otherwise go undetected)
+* **Audit** - This section tracks audit logs help you determine who is accessing your Artifactory instance and from where. These can help you track potentially malicious requests or processes (such as CI jobs) using expired credentials.
```
```suggestion
* **Audit** - This section tracks audit logs that help you determine who is accessing your Artifactory instance and from where. These can help you track potentially malicious requests or processes (such as CI jobs) using expired credentials.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+This dashboard is divided into three sections Application, Audit and Requests
+* **Application** - This section tracks Log Volume(information about different log sources) and Artifactory Errors over time(bursts of application errors that may otherwise go undetected)
+* **Audit** - This section tracks audit logs help you determine who is accessing your Artifactory instance and from where. These can help you track potentially malicious requests or processes (such as CI jobs) using expired credentials.
+* **Requests** - This section tracks HTTP response codes, Top 10 IP addresses for uploads and downloads
```
```suggestion
* **Requests** - This section tracks HTTP response codes and the top 10 IP addresses for uploads and downloads.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+* **Requests** - This section tracks HTTP response codes, Top 10 IP addresses for uploads and downloads
+
+#### JFrog Artifactory Metrics dashboard
+This dashboard tracks Artifactory System Metrics, JVM memory, Garbage Collection, Database Connections, and HTTP Connections metrics
```
```suggestion
This dashboard tracks Artifactory System Metrics, JVM memory, Garbage Collection, Database Connections, and HTTP Connections metrics.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+This dashboard tracks Artifactory System Metrics, JVM memory, Garbage Collection, Database Connections, and HTTP Connections metrics
+
+#### JFrog Xray Logs dashboard
+Thi dashboard provides a summary of access, service and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight
```
```suggestion
This dashboard provides a summary of access, service and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+Thi dashboard provides a summary of access, service and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight
+
+#### JFrog Xray Violations dashboard
+This dashboard provides an aggregated summary of all the license violations and security vulnerabilities found by Xray. Information is segmented by watch policies and rules. Trending information is provided on the type and severity of violations over time, as well as, insights on most frequently occurring CVEs, top impacted artifacts and components.
```
```suggestion
This dashboard provides an aggregated summary of all the license violations and security vulnerabilities found by Xray. Information is segmented by watch policies and rules. Trending information is provided on the type and severity of violations over time, as well as insights on most frequently occurring CVEs, top impacted artifacts and components.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+This dashboard provides an aggregated summary of all the license violations and security vulnerabilities found by Xray. Information is segmented by watch policies and rules. Trending information is provided on the type and severity of violations over time, as well as, insights on most frequently occurring CVEs, top impacted artifacts and components.
+
+#### JFrog Xray Metrics dashboard
+This dashboard tracks System Metrics, and data metrics about Scanned Artifacts and Scanned Components
```
```suggestion
This dashboard tracks System Metrics and data metrics about Scanned Artifacts and Scanned Components.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
````diff
@@ -0,0 +1,273 @@
+```
+
+
+### JFrog dashboards
````
I would move this section to the Overview section of the readme.

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+
+### Troubleshooting
+
+**Q : I have JFrog Cloud Enterprise + license. Will I be able to use this integration?**
```
```suggestion
**Q : I have a JFrog Cloud Enterprise + license. Will I be able to use this integration?**
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+
+**Q : I have JFrog Cloud Enterprise + license. Will I be able to use this integration?**
+
+A: No.This integration is only built for JFrog customers who have on-prem or self-hosted JFrog subscription. JFrog is working on a SaaS Log streaming solution which will allow our SaaS customers to stream logs to Datadog. We hope to launch that solution in late 2023.
```
```suggestion
A: No. This integration is only built for JFrog customers who have the on-prem or self-hosted JFrog subscription. JFrog is working on a SaaS Log streaming solution which will allow our SaaS customers to stream logs to Datadog. We hope to launch that solution in late 2023.
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+
+A: No.This integration is only built for JFrog customers who have on-prem or self-hosted JFrog subscription. JFrog is working on a SaaS Log streaming solution which will allow our SaaS customers to stream logs to Datadog. We hope to launch that solution in late 2023.
+
+**Q : I am about to upgrade from on-prem to JFrog Cloud. Can I expect all the same logs to stream into Datadog from my SaaS instance post migration when I install the SaaS version of the integration?**
```
```suggestion
**Q : I am about to upgrade from on-prem to JFrog Cloud. Can I expect all the same logs to stream into Datadog from my SaaS instance post-migration when I install the SaaS version of the integration?**
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,273 @@
+
+**Q : I am about to upgrade from on-prem to JFrog Cloud. Can I expect all the same logs to stream into Datadog from my SaaS instance post migration when I install the SaaS version of the integration?**
+
+A: At launch, the SaaS version of the integration will only stream the following 3 logs from your SaaS JFrog instance to Datadog
```
This sentence is incomplete

anawishnoff commented on jfrog_platform_self-hosted/assets/dashboards/jfrog_artifactory_logs_(self-hosted).json:
It looks like this dashboard is has a screenboard layout. Can you please convert all dashboards to a dashboard layout?

anawishnoff commented on jfrog_platform_self-hosted/manifest.json:
```diff
@@ -0,0 +1,81 @@
+        "caption": "JFrog Artifactory Logs dashboard",
+        "image_url": "images/jfrog_artifactory_logs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Logo",
+        "image_url": "images/JFrog+Logo.png"
+      },
```
```suggestion
```
The logo does not need to be added here, it will automatically be added to your tile.

anawishnoff commented on jfrog_platform_self-hosted/manifest.json:
```diff
@@ -0,0 +1,81 @@
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Metrics dashboard",
+        "image_url": "images/jfrog_xray_metrics.png"
```
Please ensure all images fit our [media guidelines](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel) - specifically, all images must have a 16:9 aspect ratio.

anawishnoff commented on jfrog_platform_self-hosted/manifest.json:
```diff
@@ -0,0 +1,81 @@
+      "Category::Security",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```
```suggestion
      "Supported OS::macOS",
      "Offering::Integration"
```
Please also add submitted data types in the following format:
`"Submitted Data Type::Metrics"`

anawishnoff reviewed (changes requested):
Mostly wording updates and general comments.

The setup section is quite long, have you considered perhaps hosting these on a doc on your website so the user can easily navigate to the sections that are relevant to them?

nmuesch commented on jfrog_platform_self-hosted/assets/dashboards/jfrog_xray_logs_(self-hosted).json:
```diff
@@ -0,0 +1,248 @@
+                "horizontal_align": "center",
+                "sizing": "contain",
+                "type": "image",
+                "url": "https://raw.githubusercontent.com/jfrogtraining/integrations-extras/master/jfrog_platform/assets/images/xray.png",
```
Can you replace these URLs with:

```suggestion
                "url": "https://raw.githubusercontent.com/DataDog/integrations-extras/master/jfrog_platform/assets/images/xray.png",
```

This link won't work until this PR is merged, but any github links should point to the Datadog org copy instead of the fork (it looks like there's a couple cases of this)

nmuesch commented on jfrog_platform_self-hosted/CHANGELOG.md:
```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - jfrog_platform
-All changes to the JFrog platform integration will be documented in this file.
+All changes to the JFrog Platform (Self-hosted) integration will be documented in this file.
```
Non-blocking nit: This comment was probably added a while back. We actually now show the CHANGELOG directly in the in-app tile! So this sentence can probably be removed since it doesn't make sense in all contexts where this data is shown.

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,78 @@
+* Install the JFrog Platform (Self-hosted) integration.
+
+### Fluentd Installation
+We recommend following the installation guide that matches your environment
```
```suggestion
We recommend following the installation guide that matches your environment:
```

anawishnoff commented on jfrog_platform_self-hosted/README.md:
```diff
@@ -0,0 +1,78 @@
+
+**Q : I am about to upgrade from on-prem to JFrog Cloud. Can I expect all the same logs to stream into Datadog from my SaaS instance post-migration when I install the SaaS version of the integration?**
+
+A: At launch, the SaaS version of the integration will only stream the artifactory-request, access-audit and access-security-audit logs from your SaaS JFrog instance to Datadog
```
```suggestion
A: At launch, the SaaS version of the integration will only stream the artifactory-request, access-audit and access-security-audit logs from your SaaS JFrog instance to Datadog.
```

anawishnoff commented on jfrog_platform_self-hosted/manifest.json:
```diff
@@ -0,0 +1,78 @@
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics"
```
```suggestion
      "Submitted Data Type::Metrics",
      "Submitted Data Type::Logs"
```

anawishnoff reviewed (approved):
Thanks for making all of these updates! This looks good to me. Once all approvals are in and the log pipeline is complete we can merge :)

alopezz reviewed (changes requested):
Please rebase against master.

Apart from that:

- Can anyone provide context on why it was necessary to rename `jfrog_platform` to `jfrog_platform_self-hosted`?
- Why have tests been dropped?

MahithaB commented:
@alopezz 

1. The reason jfrog_platform is renamed to jfrog_platform_self-hosted is because this integration concentrates on self-hosted versions and we're working on a new integration that covers the Saas versions
2. Tests have been dropped because we moved away from using datadog-agent to gather any information.

alopezz commented:
> 2. Tests have been dropped because we moved away from using datadog-agent to gather any information.

In that case, you should drop the implementation as well, right?

MahithaB commented:
@alopezz we've set a deprecation notice that the implementation for the existing customers will be in place till the end of August. And we've removed it from the current setup instructions already

anawishnoff commented on jfrog_platform_self-hosted/CHANGELOG.md:
```diff
@@ -1,4 +1,9 @@
-# CHANGELOG - jfrog_platform
+# CHANGELOG - jfrog_platform_self-hosted
+
+## [1.3.0] - Jun 26th, 2023
+* Modified installation steps for sending logs, metrics and violations data to Datadog
+* Supporting only OS/VM, docker and k8s forms of installation for JFrog Platform
+* Updated documentation
```
Please update the changelog to use this format:

```suggestion
# CHANGELOG - JFrog Platform Self-Hosted

## 1.3.0 / 2023-08-11

***Fixed***:
* Modified installation steps for sending logs, metrics and violations data to Datadog
* Supporting only OS/VM, docker and k8s forms of installation for JFrog Platform
* Updated documentation

```

anawishnoff replied:
This should resolve your validation issues!

MahithaB replied:
@anawishnoff I've updated the CHANGELOG and the validate repository is still not successful 😢

nmuesch replied:
Hey @MahithaB There was indeed an issue with `ddev` that got fixed and released. I've re-run the CI and the latest failure is vaild: its failing because the CODEOWNERS file will need to be updated to reflect the new integration folder name. Once thats updated the CI should pass again. Sorry for the inconvenience there!

MahithaB replied:
@nmuesch thank you. I've changed the CODEOWNERS file to the new integration folder name. I think there is a new error now. Can you please give me more information on that?

nmuesch replied:
@MahithaB The service_checks.json file is required even if its just an empty json object. Can you put that file back and add this to the manifest:

```
      "service_checks": {
        "metadata_path": "assets/service_checks.json"
      }
```

MahithaB replied:
@nmuesch Got it, added the service_checks back

MahithaB replied:
@nmuesch  Can you give info the next validation issue

nmuesch replied:
Looks like the folder rename requires the codecov file to need to be updated. From the error it looks like you can run `ddev validate ci --sync` and commit the changes to get past it

MahithaB replied:
@nmuesch can I get more info on the next error please

nmuesch replied:
This one looks in Ofeks territory so I may not be able to help with specifics. But I think it's because the configuration spec.yaml file is deleted and not in the new folder.

MahithaB replied:
@ofek can I get more information on the current error? Thanks!

ofek reviewed (changes requested):
Hello! The root directory needs to be renamed from `jfrog_platform_self-hosted` to `jfrog_platform_self_hosted` and the directory under `datadog_checks` needs to be named the same

MahithaB commented:
@ofek I've renamed the root directory, and the directory inside datadog_checks to jfrog_platform_self_hosted. Also renamed the CODEOWNERS file to reflect the same. Thanks!

ofek commented:
You will need to reintroduce this file https://github.com/DataDog/integrations-extras/blob/master/jfrog_platform/assets/configuration/spec.yaml

More info: https://datadoghq.dev/integrations-core/meta/config-specs/#schema

MahithaB commented:
@ofek I've added the spec.yaml file back and the validate repository still fails. Can you please give more information on the error? Thanks!

anawishnoff commented on jfrog_platform_self_hosted/README.md:
```diff
@@ -0,0 +1,78 @@
+
+### Data Collected
+
+#### Metrics
```
```suggestion
### Metrics
```

anawishnoff reviewed (changes requested):
After reviewing the tile, I just had a few last minor comments. Once these are addressed and the log pipeline is complete, this should be good to go. Thank you for all of your hard work here!

nmuesch commented:
Hey @MahithaB Looks like all required reviews are in and everything is looking good! 

There just seems to be some merge conflicts that need to be addressed before we're able to merge. Could you please resolve those?

emarsha94 commented on jfrog_platform_self_hosted/README.md:
```diff
@@ -0,0 +1,75 @@
+<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. The agent check will be deprecated on August 31st 2023.</div>
```
@MahithaB This line needs to be updated to the new date that Shailendra agreed on.
